sentinel-adapter/sentinel-spring-webmvc-adapter/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>sentinel-adapter</artifactId>
+        <groupId>com.alibaba.csp</groupId>
+        <version>1.7.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>sentinel-spring-webmvc-adapter</artifactId>
+
+    <properties>
+        <java.source.version>1.8</java.source.version>
```

Comment:
It's better if we could support JDK 1.7 here, as the minimum target JDK version of Sentinel is 1.7.

Reply:
I will modify.